src/java/edu/stanford/muse/util/Messages.java

```diff
@@ -1,26 +1,92 @@
 package edu.stanford.muse.util;
 
+import edu.stanford.muse.index.Archive;
+import edu.stanford.muse.index.ArchiveReaderWriter;
+import edu.stanford.muse.webapp.ModeConfig;
+
 import java.text.MessageFormat;
 import java.util.Date;
 import java.util.Locale;
 import java.util.ResourceBundle;
 
+
 public class Messages {
 
+	//static Locale local= new Locale("fr", "FR");
+
+
+	public static String test(String archiveID, String bundleName, String key){
```

Comment:
why is this method called test?
There should be no system.out.printlns in this class.
Please add javadoc or some documentation at least for each public method.